docs/Developer Resources/Custom Packages.rst

```diff
@@ -183,6 +202,17 @@ Building deb-based DKMS and user packages can be done as follows:
    $ make -j1 deb-utils deb-dkms
    $ for file in *.deb; do sudo gdebi -q --non-interactive $file; done
 
+Starting from openzfs-2.2 release, native deb-based DKMS and user
+packages can be built as follows:
+
+.. code:: sh
+
+   $ sudo apt-get install dkms
+   $ cd zfs
+   $ ./configure
+   $ make native-deb-utils
```

Comment:
Does it miss `native-dev-dkms`?

Reply:
Nvm, I've found this: https://github.com/openzfs/zfs/blob/f6f215f07f05224baff601729be7d00a64440d0d/config/deb.am#L1-L2

Reply:
And found out that the DKMS package does not need to be "native".